tests/copilot_helper/fixtures/make_addons/expected/web/addons/subscription-filter.yml

```diff
@@ -0,0 +1,20 @@
+# SubscriptionFilter requires the Destination ARN of the CloudWatch Logs in the main archive logs account.
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+Resources:
+  SubscriptionFilter:
+    Type: AWS::Logs::SubscriptionFilter
+    Properties:
+      RoleArn: !Sub "arn:aws:iam::${AWS::AccountId}:role/CWLtoSubscriptionFilterRole"
+      LogGroupName: !Sub "/copilot/${App}/${Env}/${Name}"
+      FilterName: !Sub "${App}-${Env}-stream"
+      FilterPattern: ""
+      DestinationArn: "arn:aws:logs:eu-west-2:812359060647:destination:cwl_log_destination"
```

Comment:
Do we need to add a comment to say this account number corresponds to the Log Archive AWS account explicitly? Maybe bringing the comment at the top to here instead(?).

Reply:
yes I did think about putting it there, you are right I will move it